hoverxref-intersphinx.rst

```diff
@@ -0,0 +1,114 @@
+.. post:: June 10, 2021
+   :tags: czi, grant, czi-grant, embed, sphinx-hoverxref
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      New release of sphinx-hoverxref, our Sphinx extension to add tooltips to cross references,
+      with support for intersphinx.
+
+New release of sphinx-hoverxref with support for intersphinx
+============================================================
+
+We have released version 0.6b1 of `sphinx-hoverxref`_,
+a Sphinx extension that shows a floating window
```

Comment:
I feel like we should define a standard name for this. Wikipedia calls it `Page previews` which seems like a reasonable name to copy. 

I think `preview` is probably useful to call this instead of floating window, since it has more sematic meaning. You're actually getting to "preview" or "reference" the content without leaving the page, whereas floating window is a technical description.

Reply:
Good point. This is something to consider for the `sphinx-hoverxref` documentation as well.

Reply:
Yea, definitely should backport these changes there.

Reply:
Yeah, I've been there already: https://sphinx-hoverxref.readthedocs.io/en/latest/usage.html#id2 🙈